web/modules/custom/openideal_idea/openideal_idea.module

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Contains openideal_idea.module.
+ */
+
+use Drupal\config_pages\Entity\ConfigPages;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function openideal_idea_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (!in_array($form_id, ['node_idea_edit_form', 'node_idea_form'])) {
+    return;
+  }
+
+  // Get settings page for default challenge entity.
+  $openideal_config = config_pages_config('openideal_configurations');
+  if (
+    !empty($openideal_config)
+    && $openideal_config instanceof ConfigPages
```

Comment:
config_pages_config() will always return ConfigPages instance (or NULL if it's not saved yet).